conda/environments/cuspatial_dev_cuda11.0.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - cudf=21.06.*
   - cudatoolkit=11.0
   - gdal>=3.0.2
-  - geopandas=0.7.0
+  - geopandas>=0.9.0
```

Comment:
This should probably be constrained to `<0.10.0a0` similar to the recipe.

Reply:
I did that